torch/_dynamo/symbolic_convert.py

```diff
@@ -2445,10 +2445,22 @@ def _throw_if_in_functorch(self):
             torch._C._functorch.TransformType.Grad,
             torch._C._functorch.TransformType.Jvp,
         )
+
+        # Is there a better way to detect this?
+        from_eager = counters.get("graph_break", None) is None
```

Comment:
This is 100% wrong.

Reply:
If one of the inputs is a functorch tensor and the backend is not `eager`, raise a graph break.